tools/project_manager/diff.php

```diff
@@ -138,11 +141,25 @@
     echo "| <a href='$format_url'>$format_label</a>\n";
 }
 
+if (user_can_mentor_in_any_round()) {
+    // show option to change diff style
+    if ($bb_diffs) {
+        $bb_diffs_label = _("Show table-style diff");
+        $_GET["bb_diffs"] = "off";
+    } else {
+        $bb_diffs_label = _("Show BBCode-style diff");
+        $_GET["bb_diffs"] = "on";
+    }
+    $bb_diffs_url = "?" . attr_safe(http_build_query($_GET));
+    echo "| <a href='$bb_diffs_url'>$bb_diffs_label</a>\n";
```

Comment:
personally I get super confused when the text I see if for something I'm not currently seeing. For example, when viewing "table-style diff", the button says BB style and vice-versa. Perhaps we could use a dropdown, that shows your current mode so it is clear what you are looking at? Or at least indicate the format in a heading somewhere?

Reply:
The button actually says "Show BBCode-style diff" to make it clear that clicking the link will show BBCode, rather than being a description of what is currently shown. This is the same as the adjacent link, which says "Compare without formatting" when the formatting is being shown. Given that there is currently a checkbox for "Skip empty diffs", and a link for "Compare without formatting", using a dropdown for the BBCode style would be a third method of essentially displaying a toggle switch, all of which are in the same area of the page. 
One thing about the BBCode switch is that it's instantly obvious which style of diffs you are looking at, since they are visually so different. That means it's different to the "without formatting" or the "skip empty diffs" one, where you might not notice which way the flag is set. 
I'm happy to go with the majority though. It might also be worth bearing in mind that the target audience hasn't seen it in the current form yet, so may have some usability suggestions.